src/file/FileUtils.js

```diff
@@ -526,6 +531,23 @@ define(function (require, exports, module) {
         return pathArray.join("/");
     }
 
+    /**
+     * @param {string} ext extension string a file
+     * @return {string} returns true If Brackets can open file with given extension.
+     *
+     */
+    function canBracketsOpenFileWithGivenExtensions(ext) {
+        return !extListNotSupportedInBrackets.includes(ext);
+    }
+
+    /**
+     * @param {string} ext File Extensions to be added Not Supported List
+     *
+     */
+    function addExtensionToNotSupportedList(ext) {
```

Comment:
This feels a bit unnecessary since only `xd` will be added to the list and added functionality will never be used again.

Reply:
Fixed